### DISCUSSION TITLE
How to disable League Starter in leveling tracker ?

### DISCUSSION THREAD

**Discussion by SebaFR7474** [Lailloken/Exile-UI#399] [Q&A] [answered]
Hi, i searched in the app and wiki, but i can't find to disable league starter info, when we use the leveling tracker an a alt character ?

Thanks

**Comment by Lailloken** [comment, marked as answer]
Do you mean stuff like lab trials and library quest? You have to disable that in exile-leveling before exporting.

**Comment by SebaFR7474** [comment]
Ho ok, thanks a lot.
Thought de could choose to use it or not in the tracker.

Thanks
